src/app/global-vars.service.ts

```diff
@@ -18,11 +18,14 @@ export class GlobalVarsService {
   hideGoogle = false;
   hideJumio = false;
 
+  callback: URL | null = null;
+  isCallbackValid = true;
```

Comment:
Let's default this to false and set it to true when we successfully parse the callback